### ISSUE TITLE
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

### ISSUE THREAD

**Issue by slucha** [yatish27/linkedin-scraper#73]
Great gem, exactly what I have been looking for.
I get the following error when trying to install the gem through the command
gem install linkedin-scraper -v 1.0.4, gem install linkedin-scraper or in my gemfile with gem 'linkedin-scraper'
Seems like it has a problem on the dependency of unf_ext

`Installing unf_ext 0.0.7.2 with native extensions

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

/home/jan/.rbenv/versions/2.2.3/bin/ruby -r ./siteconf20160324-3908-iq7116.rb extconf.rb
checking for main() in -lstdc++... no
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
compiling unf.cc
make: g++: Command not found
make: *** [unf.o] Error 127

make failed, exit code 2

Gem files will remain installed in /home/jan/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/unf_ext-0.0.7.2 for inspection.
Results logged to /home/jan/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/extensions/x86-linux/2.2.0-static/unf_ext-0.0.7.2/gem_make.out
An error occurred while installing unf_ext (0.0.7.2), and Bundler cannot continue.
Make sure that gem install unf_ext -v '0.0.7.2' succeeds before bundling.`

Any idea what the pronlem could be?
Thank you

**Comment by yatish27** [comment]
Which OS are you on ?

**Comment by yatish27** [comment]
is gcc installed ?

**Comment by slucha** [comment]
Yes gcc is installed. I got it working now by ubdating the build essentials. Thak you!
I am a real newb, thanks for your patience. I get the gem working from the command line by requesting
`linkedin-scraper http://www.linkedin.com/in/jeffweiner08`

After I retried today I suddenly get the following error:

`/home/jan/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/linkedin-scraper-1.0.5/bin/linkedin-scraper:10:in `<top (required)>': uninitialized constant JSON (NameError)
	from /home/jan/.rbenv/versions/2.2.3/bin/linkedin-scraper:23:in `load'
	from /home/jan/.rbenv/versions/2.2.3/bin/linkedin-scraper:23:in `<main>'
`

Any ideas. Thank you, appretiate it a lot for taking your time.
